R/get.R

```diff
@@ -59,7 +59,7 @@ get_users <- function(src, page_size = 500, prefix = NULL, limit = Inf) {
 #' @param page_size the number of records to return per page (max 500)
 #' @param prefix Filters groups by prefix (group name).
 #' The filter is case insensitive.
-#' @param limit The max number of groups to return
+#' @param limit The max number of pages to return.
```

Comment:
Hmm, `limit` is typically about result count, not pages. If the paginated requester is not respecting `limit` correctly, we should fix that instead of documenting surprising behavior.

Reply:
Yeah, in the moment, @schloerke were experimenting with `get_groups()` and thought that the documentation was wrong. Digging deeper into the code, it just looks like there is a bug in the pagination… implementation? I need to dig into it, because it looks like it might have been around for a while. Tracked by #319, and I think perhaps high enough priority for me to look at it next week.

In the meantime I'll revert this; thanks!